source/main/gui/OverlayWrapper.cpp

```diff
@@ -62,6 +62,8 @@
 
 using namespace Ogre;
 
+static bool m_is_scaled = false; // ugly

```

Comment:
`m_` is reserved for class member variables. I use `g_` for module-scope variables.

However, why a static var when you're in a class? And you even treat the variable like member. And it's prefixed `m_`. That's uber-confusing, which isn't a problem, but there's no reason for it, and that's a problem.

Otherwise, the solution is completely fine.